docs/content/ingestion/batch-ingestion.md

```diff
@@ -157,6 +157,7 @@ Is a type of inputSpec that expects data to be laid out in a specific path forma
 |dataGranularity|Object|specifies the granularity to expect the data at, e.g. hour means to expect directories `y=XXXX/m=XX/d=XX/H=XX`.|yes|
 |inputPath|String|Base path to append the expected time path to.|yes|
 |filePattern|String|Pattern that files should match to be included.|yes|
+|pathFormat|String|Joda date-time format for each directory.|no|
```

Comment:
thanks! do you mind adding a link to http://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html for reference and also clarify what the default value is?